app/serializers/atmosphere/user_serializer_ext.rb

```diff
@@ -0,0 +1,8 @@
+module Atmosphere::UserSerializerExt
+  extend ActiveSupport::Concern
+
+  private
+
+  def additional_user_details(hsh)
```

Comment:
Unused method argument - `hsh`. If it's necessary, use `_` or `_hsh` as an argument name to indicate that it won't be used. You can also write as `additional_user_details(*)` if you want the method to accept any arguments but don't care about them.